modules/gdnative/gdnative/rid.cpp

```diff
@@ -30,45 +30,17 @@
 
 #include "gdnative/rid.h"
 
-#include "core/io/resource.h"
+#include "core/os/memory.h"
```

Comment:
Unused?

Reply:
I use this for the `memnew_placement` macro.